content/300-guides/300-upgrade-guides/200-upgrading-to-latest/10-upgrade-to-2290.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Upgrading to 2.29.0
+metaDescription: 'Manual steps when upgrading to 2.29.0.'
+---
+
+<TopBlock>
+
+This page describes manual upgrade steps that you need to perform after [upgrading to Prisma 2.29.0](.) if you want to use the `namedConstraints` Preview feature.
+
+</TopBlock>
+
+## Optional: Enable the <inlinecode>namedConstraints</inlinecode> Preview feature
+
+If you choose to enable the `namedConstraints` Preview feature, choose a workflow (Prisma Migrate or Introspection) and upgrade path that suits your requirements:
```

Comment:
2 links for Migrate + Introspection would be good here, so you can get an overview which options are meant and so you can just click them.